### DISCUSSION TITLE
Vaultwarden - Portainer / Not up to date - What am I doing wrong?

### DISCUSSION THREAD

**Discussion by gnrmarcel** [dani-garcia/vaultwarden#4306] [Q&A]
I use Vaultwarden with Docker / Portainer and it works fine. Now I'm trying to update Vaultwarden, because according to "Diagnostics" I'm still using server version 1.27.0.

If I stop the container now and click on "Recreate" in Portainer, the container is recreated. The image available is "vaultwarden/server:latest".

After starting the "new" container, the server version is still 1.27 and not 1.30.3.

What am I doing wrong? I thought that the "Recreate" function really takes the last server version and thus creates the container with the server version 1.30.3.

**Comment by nate-moo** [comment]
It works fine for me, have you tried purging unused images? or you can try checking the pull new image option
![image](https://github.com/dani-garcia/vaultwarden/assets/40650681/71976669-68cb-46f9-9809-0dd24d29ec62)
